labs/3.md

```diff
@@ -1,9 +1,30 @@
+<a id="markdown-federation-introduction" name="federation-introduction"></a>
+## Federation Introduction
+
+KubeFed identifies two cluster types:
+
+1. **Host Cluster**: Cluster where the KubeFed Controller Manager is running
+2. **Member Cluster**: Cluster being managed by KubeFed Controller Manager
+
+> **NOTE:** A Host Cluster can be a Member Cluster as well, in fact, we are going to use this approach in our lab
+
+KubeFed can be configured to work in two different ways:
+
+1. **Cluster Scoped**: KubeFed can potentially manage all objects across our clusters
+2. **Namespace Scoped**: KubeFed can manage only the content of a given namespace across our clusters
+
+When to use one or another it's a good question and the answer will depend on what your use case is. As a rule of thumb
+we will use Cluster Scoped when we want to manage not-namespaced objects like `ClusterRoles`, `ClusterRoleBindings`, `SecurityContextConstraints`, etc. 
+
+Keep in mind that _Namespace Scoped_ Federation requires a KubeFed Controller Manager on each Namespace you want to federated.
+
 <a id="markdown-deploy-federation" name="deploy-cluster-scoped-federation"></a>
 ## Deploy Cluster Scoped Federation
 
-Federation target clusters do not require federation to be installed on them at
-all, but for convenience, we are using one of the clusters (`cluster1`) to host
-the KubeFed control plane.
+We do not need to deploy KubeFed on the clusters we want to manage, we could deploy KubeFed in an cluster external to our Federation environment for example. In this lab, for convenience, we are using one of the clusters (`cluster1`) to host the KubeFed Control Plane.
+
```

Comment:
Maybe say one of the clusters ('cluster1') as the host cluster  

Just thinking that since you introduced the term above